### DISCUSSION TITLE
Duration longer than gpx track

### DISCUSSION THREAD

**Discussion by turfutils** [gpx-animator/gpx-animator#680] [Q&A]
I'm right now rendering a gpx track and can see on the preview that it has ended the job arriving at goal after 52% of the render duration. When I look in the GPX file, the first and last time is as follows:

    <time>2024-03-17T19:51:35Z</time>
    <time>2024-03-17T21:04:59Z</time>

Which is fairly accurate. At end of render the video says 17 mars 2024 23:05:00 as current time.

It seems the times in the file is GMT while the displayed time is GMT+1 which also should be correct. But the duration is amost double of the real duration or exactly one hour too much. Is this related to timezones? Expected duration: 1:13 hours, video duration 2:13 hours. I'm in GMT+1 (wintertime).

**Comment by McPringle** [comment]
GPX Animator has a lot of settings. I can't answer your question without you providing all your settings from your rendering.

Please go to "File > Save as" to save your settings to a file. Please add your GPX file(s), too, so I can reproduce your problem. To attach these files here at GitHub, you have to create a ZIP file. You can use a password to encrypt your ZIP file and send the password to marcus@fihlon.swiss (I'm the maintainer of GPX Animator). If you have problems uploading the files here, you can send them via email, too. Please mention "GPX Animator Issue #680" in the subject so I can assign the password / files to the correct issue.
